go_modules/spec/dependabot/go_modules/file_updater_spec.rb

```diff
@@ -148,5 +162,91 @@
         expect(updated_files.find { |f| f.name == "go.sum" }).to_not be_nil
       end
     end
+
+    context "vendoring" do
+      let(:project_name) { "vendor" }
+      let(:vendor) { true }
+
+      let(:dependency_name) { "github.com/pkg/errors" }
+      let(:dependency_version) { "v0.9.1" }
+      let(:dependency_previous_version) { "v0.8.0" }
+      let(:requirements) do
+        [{
+          file: "go.mod",
+          requirement: dependency_version,
+          groups: [],
+          source: {
+            type: "default",
+            source: "github.com/pkg"
+          }
+        }]
+      end
+      let(:previous_requirements) do
+        [{
+          file: "go.mod",
+          requirement: dependency_previous_version,
+          groups: [],
+          source: {
+            type: "default",
+            source: "github.com/pkg"
+          }
+        }]
+      end
+
+      it "updates the version" do
```

Comment:
```suggestion
      it "updates the go.mod" do
```

Nit: I think that's closer to what it's test is up to?